server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/aggregators/numerictype/StarTreeNumericType.java

```diff
@@ -20,7 +20,10 @@ public enum StarTreeNumericType {
     HALF_FLOAT(IndexNumericFieldData.NumericType.HALF_FLOAT, StarTreeNumericTypeConverters::halfFloatPointToDouble),
```

Comment:
https://opensearch.org/docs/latest/field-types/supported-field-types/numeric/

Byte is missing. And scaled float is also documented as numeric type though not part of number mapper - can we check

Reply:
have handled for byte. The thing with scaled_float is, it is dependent on a user setting ~ `scale_factor`, which is crucial for its conversion from sortable long to scaled_float to double. 
We need to check if it is feasible to support this field type.